### ISSUE TITLE
Getting Started

### ISSUE THREAD

**Issue by avickers** [open-austin/data-portal#14]
So we discovered last night that it doesn't properly install all the dependencies and launch if you just run the rake file.

I _think_ it's because the bundler gem is a requirement for the rakefile to do its thing.  I don't know if you can put `gem install bundler` in the rakefile itself or if it needs to be added to the readme before `rake run`.  I leave that to some ruby person to figure out. :)

**Comment by amaliebarras** [comment]
I'll add it to the readme for now. It worked once you ran `gem install bundler` though?

**Comment by avickers** [comment]
Well, last night I installed the Jekyll gem directly, then I had to start the server manually, and it worked.   That didn't work for @djake, though, who kept getting errors about some other gem even though we both had the same version of ruby installed.  I'm going to guess that I just happened to have that dependency installed on my machine already.

I can tell you that running the rake again with bundler caused a lot of gems to be installed/updated on my system, and the server actually launches automatically now.
